src/cpp/session/modules/SessionHelp.R

```diff
@@ -398,6 +398,15 @@ options(help_type = "html")
       utils::browseURL(url)
 })
 
+.rs.addJsonRpcHandler("get_vignette_title", function(topic, package){
+   .rs.tryCatch(
```

Comment:
Dealing with errors in the `onError()` is no good ? 

```java
            @Override
            public void onError(ServerError error)
            {
                Label notFound = new Label("No vignette found");
                notFound.setStyleName(styles_.warning());
                notFound.addStyleName(ConsoleResources.INSTANCE.consoleStyles().promptFullHelp());
                panel.add(notFound);
            }
```

Reply:
Good question... I _think_ this is normally used for errors that could occur in server <-> client communication, rather than errors that happen when the RPC is executed, but that's worth verifying.